public/locales/en/dataset.json

```diff
@@ -22,8 +22,17 @@
     "uploadFiles": "Upload Files"
   },
   "alerts": {
-    "draftVersion": "<b>This draft version needs to be published</b>. When ready for sharing, please <b>publish</b> it so that others can see these changes",
-    "requestedVersionNotFound": "Info – Version {0} was not found. This is version {1}",
-    "unpublishedDataset": "Privately share this dataset before it is published: {0}"
+    "draftVersion": {
+      "heading": "This draft version needs to be published",
+      "alertText": "When ready for sharing, please <b>publish</b> it so that others can see these changes"
+    },
+    "requestedVersionNotFound": {
```

Comment:
This might be a case where we want to do something different from JSF - I think it makes sense to always show the alert if the version requested doesn't exist. I can bring it up with the team.

Reply:
We talked about it in the frontend meeting and decided to always show the alert, even if the only version is draft.  I added another translation for the slightly different text of this message.